properties/size/breakpoint.json

```diff
@@ -0,0 +1,26 @@
+{
+  "size": {
+    "breakpoint": {
+      "phone": {
+        "value": "480",
+        "comment": "viewports smaller than tablets",
```

Comment:
Nitpick: The comment on this and the "tablet" breakpoints are hard to wrap my head around. I think the breakpoint names clarify enough, but if we think the comments are important it may be good to change them.